ceph/ceph_admin/bootstrap.py

```diff
@@ -143,29 +249,17 @@ def bootstrap(self: CephAdmProtocol, config: Dict) -> None:
             check_ec=True,
         )
 
-        logger.info("Bootstrap output : %s", out.read().decode())
-        logger.error("Bootstrap error: %s", err.read().decode())
+        out, err = out.read().decode(), err.read().decode()
+        logger.info("Bootstrap output : %s", out)
+        logger.error("Bootstrap error: %s", err)
 
         self.distribute_cephadm_gen_pub_key()
 
         # Verification of arguments
-
         args = self.config.get("args", {})
         for key, value in args.items():
-            func = self.get_method(key)
+            func = fetch_method(key)
             if not func:
                 continue
-            if not func(value):
+            if not func(self, value, out + err):
```

Comment:
Would it possible to add a note on why we are concatenating `out + err`

Reply:
done